lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -8,9 +8,10 @@ class Pod < KubernetesResource
       Evicted
       Preempting
     )
+    attr_writer :stream_logs
```

Comment:
do you have a use case in mind where you'd create the pod w/ `stream_logs: false` but change it before pod completed?